changelog.txt

```diff
@@ -1,3 +1,17 @@
+4.24.0
+============
+
+New Features:
+**Performance improvement by reducing the number of calls to GetAllAccessTokens()**. See issue [#2204](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2204) for details.
```

Comment:
I think this would be a bug fix, but idk.

Reply:
yes, or fundamental work. It's not a new feature.